### PR TITLE
Implement work items as vars and style suggestions

### PR DESCRIPTION
**Pull request by kylie-bee**
Primarily this change implements using a work item to select which site secret to use, but also this change includes several changes to match portal bot styling.

### CODE REVIEW

File: tasks.robot [hunk 1]
```diff
@@ -1,49 +1,73 @@
...
     RETURN    ${secret}
 
 Create password
-    [Documentation]    uses FakerLibrary to generate a new random password
-    ...    if you do not need special characters, upper, lower, etc set those to False
+    [Documentation]
+    ...    Uses FakerLibrary to generate a new random password of the provided
+    ...    ``length`` (defaults to 10 characters long). if you do not need special
+    ...    characters, upper, lower, etc set those to False
+    [Arguments]    ${length}=10    ${special_chars}=${True}    ${digits}=${True}
```
Comment (ericHanson13) [inline tasks.robot:46]: can you switch it to use work items instead?

**Review by ericHanson13 — CHANGES_REQUESTED** [review]
See comments in the comment of the code.

File: tasks.robot (hunk 1 above)
Comment (kylie-bee) [reply tasks.robot:46]: Okay, new commit I just pushed fixes this.

**Review by ericHanson13 — APPROVED** [review]

**Comment by ericHanson13** [comment]
Improvements to flow based on Blake's review.

**Pull request merged**